### PR TITLE
Improve see-also references for gateway api

### PR DESCRIPTION
**Pull request by frankbu**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/en/blog/2022/getting-started-gtwapi/index.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ $ kubectl wait -n sample-ingress --for=condition=ready gateway sample-gateway
 $ export INGRESS_HOST=$(kubectl get -n sample-ingress gateway sample-gateway -o jsonpath='{.status.addresses[*].value}')
 {{< /text >}}
 
-Then we can attach an [HTTPRoute](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io%2fv1beta1.HTTPRoute)
+Next, we attach an [HTTPRoute](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io%2fv1beta1.HTTPRoute)
```
Comment (ericvn) [inline content/en/blog/2022/getting-started-gtwapi/index.md:155]: attach a?

Comment (frankbu) [reply content/en/blog/2022/getting-started-gtwapi/index.md:155]: I think an is correct

**Review by ericvn — APPROVED** [review]

**Pull request merged**